### PR TITLE
Adding logger in duplicate.py

### PR DESCRIPTION
**Pull request by singla007**

### CODE REVIEW

**Comment by suhaibmujahid** [comment]
Thank you @singla007 for submitting the PR. I think you are working on this as part of https://github.com/mozilla/bugbug/issues/3322.

Could you pleas take care of the the follwoing note from the issue:
> Note: please use lazy % formatting in logging functions (more details can be found in the [pylint docs](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html))

To avoid lint failures, it is recommended to setup `pre-commit`: https://github.com/mozilla/bugbug#auto-formatting

**Comment by singla007** [comment]
Hi @suhaibmujahid , thanks for the review, I will update the code according to the suggestion.

File: bugbug/models/duplicate.py [hunk 1]
```diff
@@ -120,7 +124,7 @@ def get_labels(self):
         # Store all remaining ids
         non_duplicate_ids = list(all_ids - set(duplicate_ids))
 
-        print(f"Number of duplicate labels is: {self.num_duplicates}")
+        logger.info("Number of duplicate labels is: %s", self.num_duplicates)
```
Comment (suhaibmujahid) [inline bugbug/models/duplicate.py:127]:
```suggestion
        logger.info("Number of duplicate labels is: %d", self.num_duplicates)
```


When the value is an integer, please use `%d`. Please apply the same change where applicable.

Comment (singla007) [reply bugbug/models/duplicate.py:127]: thanks for pointing this mistake out, i will do the changes

**Review by suhaibmujahid — APPROVED** [review]
Thank you, @singla007! LGTM.

**Pull request merged**